synapse/rest/client/v2_alpha/account.py

```diff
@@ -200,7 +193,7 @@ class PasswordResetSubmitTokenServlet(RestServlet):
     """Handles 3PID validation token submission"""
 
     PATTERNS = client_patterns(
-        "/password_reset/(?P<medium>[^/]*)/submit_token/*$", releases=(), unstable=True
+        "/password_reset/(?P<medium>[^/]*)/submit_token$", releases=(), unstable=True
```

Comment:
```suggestion
        "/password_reset/(?P<medium>[^/]*)/submit_token$", releases=()
```

Since we're using unstable flags, we can use `r0` for the endpoints.